synapse/storage/databases/main/devices.py

```diff
@@ -2118,6 +2118,29 @@ def _add_device_outbound_poke_to_stream_txn(
                 },
             )
 
+    async def mark_redundant_device_lists_pokes(
+        self,
+        user_id: str,
+        device_id: str,
+        room_id: str,
+        converted_upto_stream_id: int,
+    ) -> None:
+        """If we've calculated the outbound pokes for a given room/device list
+        update, mark any subsequent changes as already converted"""
+
+        sql = """
+            UPDATE device_lists_changes_in_room
+            SET converted_to_destinations = true
+            WHERE stream_id > ? AND user_id = ? and device_id = ? AND room_id = ?;
```

Comment:
```suggestion
            WHERE stream_id <= ? AND user_id = ? and device_id = ? AND room_id = ?;
```

name like 'up to stream_id' sounds like it meant this...